platform-emf/src/main/java/com/softicar/platform/emf/entity/table/column/handler/EmfDisplayStringToTextAreaColumnHandler.java

```diff
@@ -12,9 +12,9 @@ public class EmfDisplayStringToTextAreaColumnHandler extends EmfDataTableValueBa
 	@Override
 	public void buildCell(IEmfDataTableCell<?, IDisplayString> cell, IDisplayString value) {
 
-		DomDiv content = new DomDiv();
+		var content = new DomDiv();
 		content.addCssClass(EmfCssClasses.EMF_DISPLAY_STRING_TO_TEXT_AREA_COLUMN_HANDLER);
-		DomTextArea textArea = new DomTextArea(value.toString(), 3, 40);
+		var textArea = new DomTextArea(value.toString()).setSize(3, 40);
```

Comment:
Kind of unrelated... but I think those numbers should be constants.

Reply:
- I see not big benefit in making them constants, these numbers have no "semantics" and are not reused.
- Actually, we should probably remove all those calls and make this over CSS.